.setup/install_system.sh

```diff
@@ -208,6 +208,7 @@ source ${CURRENT_DIR}/distro_setup/setup_distro.sh
 #libraries for QR code processing:
 #install DLL for zbar
 apt-get install libzbar0 --yes
+apt-get install python3-opencv --yes
```

Comment:
You don't want to install a python dependency with both `apt-get` and `pip` as they can end up colliding in weird ways.

Reply:
ah ok, I didn't realize there could be a collision between them, since the pip package is an unofficial pip distribution of opencv and the recommended way seems to be via the apt install method I'll remove it from the system_requirements.txt

I added this line since when I tried installing on ubuntu20 there were some OS dependencies missing and it seemed like just installing open cv via apt is the easiest way forward